Content.Client/UserInterface/Systems/Actions/ActionUIController.cs

```diff
@@ -399,7 +430,13 @@ private void OnActionsUpdated()
         QueueWindowUpdate();
 
         if (_actionsSystem != null)
-            _container?.SetActionData(_actionsSystem, _actions.ToArray());
+            UpdateActionContainer();
+    }
+
+    private void UpdateActionContainer()
```

Comment:
Personally, I don't see a need for this function. Most of the places it is used already perform a null check on _actionsSystem or use the null check operator (?) on _container